bot/exts/evergreen/snakes/_snakes_cog.py

```diff
@@ -1131,21 +1131,11 @@ async def zen_command(self, ctx: Context) -> None:
     @video_command.error
     async def command_error(self, ctx: Context, error: CommandError) -> None:
         """Local error handler for the Snake Cog."""
-        embed = Embed()
-        embed.colour = Colour.red()
-
-        if isinstance(error, BadArgument):
-            embed.description = str(error)
-            embed.title = random.choice(ERROR_REPLIES)
-
-        elif isinstance(error, OSError):
+        if isinstance(error, OSError):
```

Comment:
I realised this actually doesn't even do anything since a `CommandError`'s class hierarchy has no relation to `OSError`. It would have to use  `error.original`, if that attribute even happens to exist. And again, I'm not even sure why this has a special case for `OSError` in the first place. Does anyone else know if this is even needed?

Reply:
I am guessing this is how the original cog was designed, it doesn't seem very needed now though.

Reply:
I have a radical solution: If none of us can think of a reason for it being here after reading the code and it's not documented, we should remove it. It would get this PR going (which is important for the Advent of Code PR, which we ideally merge soon, as the event is ongoing) and, if we do that it was needed, we'll notice that at some point and can add it back in.

I think it's preferable to having some vestigial piece of code in our cog that's non-critical (it's not part of the core features), seemingly useless, has no known purpose, but is blocking development progress.

Reply:
Right, so, after some thinking, I think an `OSError` may be raised if we failed to download any bytes for an image. It would be a bit weird for this to happen, but I think `Image.open` raises an OSError for an empty BytesIO read.